.github/workflows/release.yml

```diff
@@ -108,6 +108,19 @@ jobs:
         run: ./gradlew build publishPlugins publishPluginMavenPublicationToSonatypeRepository closeAndReleaseSonatypeStagingRepository
         working-directory: gradle-plugins
 
+      - shell: bash
+        name: Collect SBOM's
+        run: |
+          mkdir sboms
+          cp javaagent/build/spdx/*.spdx.json sboms
+          zip opentelemetry-java-instrumentation-SBOM.zip sboms/*
+
+      - uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
+        name: Save SBOM's to build
+        with:
+          name: opentelemetry-java-instrumentation-SBOM
+          path: "sboms/*.json"
+
```

Comment:
why attach to the build, when we're attaching to the release just below?

Reply:
I think it's good practice to attach relevant artifacts to the build process for future reference. If this bothers you I can remove it.